cli/package.json

```diff
@@ -6,7 +6,6 @@
     "datastore-todos": "cli.js"
   },
   "dependencies": {
-    "gcloud-node-todos": "*",
```

Comment:
I think this was more `npm` was installing the module over and over in a nested fashion because it requires itself. Instead, if the CLI is part of the package, it should just require the files directly.